install_config/upgrading/automated_upgrades.adoc

```diff
@@ -329,6 +329,91 @@ $ ansible-playbook -i <path/to/inventory/file> \
 See xref:../../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[Manging
 Nodes] for more on node labels.
 
+
+[[upgrade-hooks]]
+=== Customizing Upgrades With Ansible Hooks
+
+Ansible allows for operators to execute custom tasks during specific operations
```

Comment:
It was "The ansible installer" before. This was my misinterpretation, I'll set it back to something closer to the original.